src/unix/linux-core.c

```diff
@@ -90,7 +90,11 @@ int uv__platform_loop_init(uv_loop_t* loop) {
    * a.k.a. Lollipop. Since EPOLL_CLOEXEC is an alias for O_CLOEXEC on all
    * architectures, we just use that instead.
    */
+#if defined(__ANDROID_API__) && __ANDROID_API__ < 21
+  fd = (errno = ENOSYS, -1);
```

Comment:
This inside-out style isn't wrong exactly but I'd prefer it if you wrote it as `fd = -1, errno = ENOSYS` (or as two statements) because that's a style we use elsewhere in libuv.

Reply:
Updated.